requirements_all.txt

```diff
@@ -1239,14 +1239,16 @@ piglow==1.2.4
 # homeassistant.components.pilight
 pilight==0.1.1
 
+# homeassistant.components.tensorflow
+pillow==9.0.0
```

Comment:
Should we update pillow for tensorflow too?

Reply:
yes, wth went wrong there? Will take a look :S